app/views/users/show.html.erb

```diff
@@ -1 +1,16 @@
-<h1>User ID is <%= @user %></h1>
\ No newline at end of file
+
+	<%= render 'user', user:@user	%>
+
+	<div class='bio-main-div'>
+
+
+	<h2>Bio</h2>
+		<%= @user.bio%>
+	</div>
+
+	<%= render @user.posts	%>
+
+<%= link_to user_posts_path(@user) do %>
+<button>See All Posts</button>
+<%end%>
```

Comment:
kindly use the `recent_posts` method you have defined in the **User** model to display the three recent posts of a user.  It is one of the project requirements. As you can see in the screenshot, it is displaying 4 posts, while it should only display three recent posts. 
![image](https://github.com/MosDevx/BlogApp/assets/33268927/552365aa-fd66-4bf5-bb45-74b13f49d057)

Reply:
done

Reply:
done